app/services/slack_notification_service.rb

```diff
@@ -100,12 +100,13 @@ def js_channels(pr, lang)
     return unless pr[:name]
 
     repo_name = pr[:name].downcase
+    topics = pr[:topics].present? ? pr[:topics] : []
 
-    if (repo_name.include? 'react') && (repo_name.include? 'native')
+    if ((repo_name.include? 'react') && (repo_name.include? 'native')) || (topics.include? 'react-native')
```

Comment:
WDYT about extracting these conditions into separate methods to make it easier to read?

something like: 

```
  def js_channels(pr, lang)
      if react_native_repo?(repo_name)
        ....
      end
  end
    
    def react_native_repo?(repo_name)
      ....
    end
    
```

Reply:
I refactored it but with only one method